scripts/flashida/write_method.py

```
@@ -0,0 +1,105 @@
+    {'key': 'MaxMass', 'name': 'MaxMass', 'value': 50000},
+    {'key': 'Tolerances', 'name': 'Tolerances', 'value': [10.0, 10.0]},
+    {'key': 'TargetLogs', 'name': 'TargetLogs', 'value': [r'C:\Users\KyowonJeong\Desktop\FLASHIdaTmp\test1.log']},
+    {'key': 'RTWindow', 'name': 'RTWindow', 'value': 180},
+    {'key': 'TargetMode', 'name': 'TargetMode', 'value': 0},
+    {'key': 'UseFAIMS', 'name': 'UseFAIMS', 'value': False},
+    {'key': 'UseCVQScore', 'name': 'UseCVQScore', 'value': False},
+    {'key': 'CycleTime', 'name': 'CycleTime', 'value': 180},
+    {'key': 'CVValues', 'name': 'CVValues', 'value': [-10.0, -30.0, -40.0, -50.0, -60.0]},
+]
+
+def get_params():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)